src/browser_action/vitals.js

```diff
@@ -165,9 +165,9 @@
         }
 
         // Overlay close button
-        const overlayClose = document.getElementById('web-vitals-close');
+        let overlayClose = document.getElementById('web-vitals-close');
         if (overlayClose === null) {
-          const overlayClose = document.createElement('button');
+          overlayClose = document.createElement('button');
```

Comment:
Same thing as above here.

Reply:
True, but think following the similar pattern to the `overlayElement` is probably clearer, rather going with a slightly different pattern here.

The other change we could so is just to move this whole thing into the `if (overlayElement === null) {` section as when there's no overlay, there should also be no close button.

But I say for simplicity now let's leave as is.

Reply:
OK you've convinced me. Updated.